llvm/lib/Target/X86/GISel/X86InstructionSelector.cpp

```diff
@@ -472,6 +482,10 @@ unsigned X86InstructionSelector::getLoadStoreOp(const LLT &Ty,
                     : (HasAVX512 ? X86::VMOVSDZmr :
                        HasAVX    ? X86::VMOVSDmr :
                                    X86::MOVSDmr);
+    if (X86::PSRRegBankID == RB.getID())
+      return Isload ? X86::LD_Fp64m : X86::ST_Fp64m;
+  } else if (Ty == LLT::scalar(80) || Ty == LLT::pointer(0, 80)) {
```

Comment:
Do you support 80bit pointers?

Reply:
I don't really know. This check is a result of copy paste - I should have verified if we have such pointers. I'll get back on this

Reply:
Your pointer is defined as:
https://github.com/llvm/llvm-project/blob/7c181955637197f7a1b37b00d271254f85f66d85/llvm/lib/Target/X86/GISel/X86LegalizerInfo.cpp#L47